src/ext/components/frame-output.coffee

```diff
@@ -59,7 +59,7 @@ module.exports = (_, _go, _frame) ->
     _.insertAndExecuteCell 'cs', "predict frame: #{stringify _frame.frame_id.name}"
 
   download = ->
-    window.open _.ContextPath + "3/DownloadDataset?frame_id=#{encodeURIComponent _frame.frame_id.name}", '_blank'
+    window.open _.ContextPath + "3/DownloadDataset?frame_id=#{encodeURIComponent _frame.frame_id.name}&hex_string=false&escape_quotes=true", '_blank'
```

Comment:
looks a bit magic tbh.
From `Frame.CSVStreamParams`, you're just setting the default values:
```java
  public static class CSVStreamParams extends Iced<CSVStreamParams> {
    public static final char DEFAULT_SEPARATOR = ',';
    public static final char DEFAULT_ESCAPE = '"';

    boolean _headers = true;
    boolean _quoteColumnNames = true; 
    boolean _hexString = false;
    boolean _escapeQuotes = true;
```
so something must be wrong somewhere else.

I suggest it is in `DatasetServlet`:
```java
      String hex_string = request.getParameter("hex_string");
      String escape_quotes_string = request.getParameter("escape_quotes");
      ...
      Frame.CSVStreamParams parms = new Frame.CSVStreamParams()
          .setHexString(Boolean.parseBoolean(hex_string))
          .setEscapeQuotes(Boolean.parseBoolean(escape_quotes_string));
```
request params are `null` by default,and `Boolean.parseBoolean(null) == false`, which may override the defaults defined in `CSVStreamParams`.

I'd suggest instead to do sth like :
```
    Frame.CSVStreamParams parms = new Frame.CSVStreamParams();
    if (hex_string != null) params.setHexString(Boolean.parseBoolean(hex_string));
    if (escape_quotes_string != null) params.setEscapeQuotes(Boolean.parseBoolean(escape_quotes_string));
```
and you won't need to set magic params in Flow.
By the way, it means that Py/R clients were also not using defaults when calling this download servlet.

Finally curious to know why `escape_quotes=true` changes the encoding: it probably doesn't. But the downloaded csv was just broken, correct?

Reply:
I see.. yes, without `escape_quotes=true` the csv has different number of columns and rows than as is expected / again some merging happens, but different one than what was happening during `df.as_data_frame()` in py // no chinese characters, it just gets broken

Reply:
I guess you can close this PR then, right?
The Flow fix will also go in https://github.com/h2oai/h2o-3/pull/5589